### PR TITLE
FIX: bug/1257904

### PR DESCRIPTION
**Pull request by xenolog**
puppet send non-boolean value instead true/false

### CODE REVIEW

File: deployment/puppet/l23network/lib/puppet/provider/l2_ovs_port/ovs.rb [hunk 1]
```diff
@@ -55,7 +55,8 @@ def create
       end
     end
     # enable vlan_splinters if need
-    if @resource[:vlan_splinters]
+    if @resource[:vlan_splinters].to_s() == 'true'  # fucking puppet send non-boolean value instead true/false
```
Comment (xarses) [inline deployment/puppet/l23network/lib/puppet/provider/l2_ovs_port/ovs.rb]: Please remove the f-word from the comment

**Comment by ashaposhnikov** [comment]
+1 if you'll remove f-word

**Comment by xarses** [comment]
Confirmed that it removes the problem, update the comment.

**Comment by aglarendil** [comment]
+2

**Pull request merged**